mParticleCore.brs

```diff
@@ -138,6 +138,94 @@ function mParticleConstants() as object
                                 end function
         
     }
+    '
+    ' Consent Management
+    '
+    ConsentState = {
+        build : function ()
+            consentState = {}
+            consentState.gdpr = {}
+            consentState.ccpa = {}
+            return consentState
+        end function,
+
+        ' GDPR
+        addGDPRConsentState : function (consentState as object, purpose as string, gdprConsent as object)
+            consentState.gdpr.AddReplace(purpose, gdprConsent)
+        end function,
+        getCCPAConsentState : function (consentState as object)
+            return consentState.ccpa.Lookup("data_sale_opt_out")
+        end function,
+        getGDPRConsentState : function (consentState as object, purpose as string)
```

Comment:
`getGDPRConsentState` shouldn't take any arguments. it should just return the GDPRConsentState.  it's up to the developer if they want to then lookup a purpose on it